packages/api/src/sdk-bundle-api/sdk-to-bundle/test-run-environment.ts

```diff
@@ -64,3 +64,9 @@ export interface TestRunGitHubWorkflowDispatchContext {
   /** Resolved head commit hash */
   headSha: string;
 }
+
+export type TestRunGitLabContext = TestRunGitLabPullRequestContext;
+export interface TestRunGitLabPullRequestContext {
+  type: "gitlab";
+  event: "pull-request";
+}
```

Comment:
We may also want the pull request title URL etc. if you want it to display in the UI on the test run page: 

<img width="451" alt="image" src="https://github.com/alwaysmeticulous/meticulous-sdk/assets/793481/191ac49a-1d0c-4963-b48b-15a9f47975fb">

(though I don't know if that's easy to grab from GitLab API)

Also can always add it later / this PR looks good to merge as is.

Reply:
Updated to include the fields necessary to render that component. Also included baseSha and headSha since those are both included in github pull request context.